packages/web-react/src/components/FileUploader/README.md

```diff
@@ -469,39 +469,53 @@ and a collection of props selected above for easier implementation. Other props
+| `labelText`           | `string`                                | —        | ✕        | Label for input in Drop zone                                                                                              |
+| `linkText`            | `string`                                | —        | ✕        | Link text in input in Drop zone                                                                                           |
+| `listId`              | `string`                                | —        | ✓        | FileUploaderList id                                                                                                       |
+| `listProps`           | `Partial<FileUploaderListBaseProps>`    | —        | ✕        | Rest of FileUploaderList props                                                                                            |
+| `maxFileSize`         | `number`                                | 1000000  | ✕        | The maximum size of the uploaded file in bytes. [Learn how file sizes are calculated](#understanding-file-size-in-bytes). |
+| `maxUploadedFiles`    | `number`                                | 10       | ✕        | Maximum file upload queue size                                                                                            |
+| `onChange`            | `(fileQueue: FileQueueMapType) => void` | —        | ✕        | Callback on change in fileQueue                                                                                           |
+| `onInputError`        | `FileUploaderErrorCallbackType`         | —        | ✕        | Callback on error condition                                                                                               |
+| `queueLimitBehavior`  | \[`hide` \| `disable` \| `none`]        | `none`   | ✕        | Input behavior when the file queue is filled                                                                              |
+| `validationState`     | `ValidationState`                       | —        | ✕        | Validation state                                                                                                          |
+| `validationText`      | \[`string` \| `string[]`]               | —        | ✕        | Validation status text                                                                                                    |
 
 On top of the API options, the components accept [additional attributes][readme-additional-attributes].
 If you need more control over the styling of a component, you can use [style props][readme-style-props]
 and [escape hatches][readme-escape-hatches].
 
 For detailed information see [FileUploader][file-uploader] component.
 
+### Understanding File Size in Bytes
+
+File size is measured in bytes, where larger units such as kilobytes (KB) or megabytes (MB) are derived as follows:
+
+1 KB (kilobyte) = 1,024 bytes
```

Comment:
**suggestion (if-minor):** To be technically correct, you are using KiB (kibibytes). https://en.wikipedia.org/wiki/Binary_prefix

1 KiB (kilobyte) = 1,024 bytes

This is correct in the metric system
1 KB (kilobyte) = 1,000 bytes

https://en.wikipedia.org/wiki/Metric_prefix

Maybe it would be nice also to mention the example that @vaclavbarta wrote to the ticket with the PHP in file and the `upload_max_file_size` - https://www.php.net/manual/en/ini.core.php#ini.upload-max-filesize

The same maybe should also be for the Next.js - https://nextjs.org/docs/app/api-reference/config/next-config-js/serverActions#bodysizelimit

In both projects, metric prefixes are used but the values are calculated in the binary system.